ci-operator/step-registry/optional-operators/cvp-common/scorecard-test/optional-operators-cvp-scorecard-test-ref.metadata.json

```diff
@@ -0,0 +1,9 @@
+{
+	"path": "optional-operators/cvp-common/scorecard-test/optional-operators-cvp-scorecard-test-ref.yaml",
+	"owners": {
+		"approvers": [
+			"dirgim",
+			"petr-muller"
```

Comment:
hrm, I guess we need to fix the parent OWNERS to better refect the reality (don't bother in this PR, I'll try to address this later)